parm/gfs/conTime.yaml

```diff
@@ -460,13 +460,13 @@ graphics:
       figure:
         layout: [4,1]
         figure size: [20,18]
-        tight layout:
+#       tight layout:
```

Comment:
@kevindougherty-noaa can we figure out why `tight layout` and `plot logo` aren't working on wcoss?

Reply:
@EdwardSafford-NOAA What is the error that occurs?

Reply:
Traceback (most recent call last):
  File "/lfs/h2/emc/da/noscrub/edward.safford/eva/opt/bin/eva", line 8, in <module>
    sys.exit(main())
  File "/lfs/h2/emc/da/noscrub/edward.safford/eva/opt/eva/eva_driver.py", line 120, in main
    eva(config_file)
  File "/lfs/h2/emc/da/noscrub/edward.safford/eva/opt/eva/eva_driver.py", line 85, in eva
    figure_driver(eva_dict, data_collections, timing, logger)
  File "/lfs/h2/emc/da/noscrub/edward.safford/eva/opt/eva/plotting/batch/base/plot_tools/figure_driver.py", line 150, in figure_driver
    make_figure(handler, figure_conf, plots_conf,
  File "/lfs/h2/emc/da/noscrub/edward.safford/eva/opt/eva/plotting/batch/base/plot_tools/figure_driver.py", line 256, in make_figure
    fig.save_figure(output_file, **saveargs)
  File "/lfs/h2/emc/da/noscrub/edward.safford/eva/opt/emcpy/plots/create_plots.py", line 232, in save_figure
    self.fig.savefig(pathfile, **kwargs)
  File "/apps/prod/ve/intel/19.1.3.304/python/3.10.4/evs/1.0/lib/python3.10/site-packages/matplotlib/figure.py", line 3343, in savefig
    self.canvas.print_figure(fname, **kwargs)
  File "/apps/prod/ve/intel/19.1.3.304/python/3.10.4/evs/1.0/lib/python3.10/site-packages/matplotlib/backend_bases.py", line 2366, in print_figure
    result = print_method(
  File "/apps/prod/ve/intel/19.1.3.304/python/3.10.4/evs/1.0/lib/python3.10/site-packages/matplotlib/backend_bases.py", line 2232, in <lambda>
    print_method = functools.wraps(meth)(lambda *args, **kwargs: meth(
TypeError: FigureCanvasAgg.print_png() got an unexpected keyword argument 'tight layout'

Reply:
The same issue is at play with the logos, it reports an unexpected keyword argument.